### DISCUSSION TITLE
Temp Sensor for Hot Spot RTX 4000 series

### DISCUSSION THREAD

**Discussion by DanielCurly78** [Rem0o/FanControl.Releases#2731] [Q&A]
Hey!
I just found that the Hot Spot sensor for RTX 4000 series from nVidia is not available. For RTX 3000 series it is available. I now use the sensor with the HWINFO plugin, but I'd rather prefer to use the one from FanControl. Is it possible to be added?

**Comment by Fr0stX76** [comment]
https://github.com/Rem0o/FanControl.Releases/issues/2470

**Reply by DanielCurly78** [reply]
I already did that, for 4000 series is not working :) I still can't see the Hot Spot temperature for GPU

**Reply by Fr0stX76** [reply]
Have you tried librehardwaremonitor on it's own? If it does'nt have the hot spot, then FanControl wont either. 

[Latest build](https://nightly.link/LibreHardwareMonitor/LibreHardwareMonitor/workflows/master/master/LibreHardwareMonitor-net472.zip)

It does show on my 40 series card... So it should on yours too

![image](https://github.com/user-attachments/assets/c324bfde-2dc4-47ac-aa8d-2fa2be438fbf)

**Reply by DanielCurly78** [reply]
I see the temp in Libre hardware monitor. But not in Fancontrol. Here is a picture:
![image](https://github.com/user-attachments/assets/9083dcf2-d0ce-43da-a93a-764a938feb9e)

**Reply by Fr0stX76** [reply]
Then I don't know. It's working on my system

![image](https://github.com/user-attachments/assets/bf7497dc-d003-409c-a02c-8d13e4ba8dba)
